packages/easy-coding-standard/packages/snippet-formatter/tests/Markdown/config/array_fixer.php

```diff
@@ -3,12 +3,23 @@
 declare(strict_types=1);
 
 use PhpCsFixer\Fixer\ArrayNotation\ArraySyntaxFixer;
+use PhpCsFixer\Fixer\ArrayNotation\TrailingCommaInMultilineArrayFixer;
+use PhpCsFixer\Fixer\Whitespace\ArrayIndentationFixer;
+use PhpCsFixer\Fixer\Whitespace\NoTrailingWhitespaceFixer;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\CodingStandard\Fixer\ArrayNotation\ArrayOpenerAndCloserNewlineFixer;
+use Symplify\CodingStandard\Fixer\LineLength\LineLengthFixer;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
     $services->set(ArraySyntaxFixer::class)
         ->call('configure', [[
             'syntax' => 'short',
         ]]);
+
+    $services->set(ArrayOpenerAndCloserNewlineFixer::class);
+    $services->set(LineLengthFixer::class);
```

Comment:
This one should be removed, as it skips `ArrayOpenerAndCloserNewlineFixer` and possibly hides the bug

Reply:
remove `LineLengthFixer` make error: 

```bash
--- Expected
+++ Actual
@@ @@
     $services = $containerConfigurator->services();
 
     $services->set(MoveValueObjectsToValueObjectDirectoryRector::class)
-        ->call(
-            'configure',
-            [[
-                MoveValueObjectsToValueObjectDirectoryRector::TYPES => [
-                    'ValueObjectInterfaceClassName',
-                ], MoveValueObjectsToValueObjectDirectoryRector::SUFFIXES => [
-                    'Search',
-                ], MoveValueObjectsToValueObjectDirectoryRector::ENABLE_VALUE_OBJECT_GUESSING => true
-            ]]
-        );
+        ->call('configure', [[MoveValueObjectsToValueObjectDirectoryRector::TYPES => ['ValueObjectInterfaceClassName'], MoveValueObjectsToValueObjectDirectoryRector::SUFFIXES => ['Search'], MoveValueObjectsToValueObjectDirectoryRector::ENABLE_VALUE_OBJECT_GUESSING => true]]);
 };
```

Reply:
Yes. That's the original bug we need to fix. 

Only the `ArrayOpenerAndCloserNewlineFixer` should do this:

```diff
-->call('configure', [[MoveValueObjectsToValueObjectDirectoryRector::TYPES 
+->call('configure', [[
+MoveValueObjectsToValueObjectDirectoryRector::TYPES 
```

You can try it on standalone PHP file to see it.

Reply:
This original fixture must work: https://github.com/symplify/symplify/pull/2241/files

No need to add any other fixture or any other rules.

Reply:
I reverted the fixer addition and try run ecs to `standalone-file.php`, seems not apply the fix (marked as already correct):

![Screen Shot 2020-09-26 at 4 46 37 PM](https://user-images.githubusercontent.com/459648/94337889-0ea76a80-0018-11eb-9180-12ecf1849d43.png)